code/style.css

```diff
@@ -1,5 +1,465 @@
-/* After you've linked the CSS file in the HTML,
-this should turn the background blue ;) */
+@import url('https://fonts.googleapis.com/css2?family=Cinzel:wght@400..900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
-  background: blue;
+  background-color: white;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: white;
+  padding: 20px;
+}
+
+.top-row {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 20px;
+  flex-grow: 1;
+}
+
+.nav-menu {
+  display: flex;
+  list-style: none;
+  position: relative;
+}
+
+.nav-menu li a {
+  text-decoration: none;
+  color: black;
+  padding: 8px 16px;
+  font-family: 'Cinzel', sans-serif;
```

Comment:
Looking at Cinzel it is a serif font, so maybe you would like the fallback to also be a serif and not a sans-serif?